### DISCUSSION TITLE
Improved automatic filters

### DISCUSSION THREAD

**Discussion by gortador** [eveseat/seat#761] [Q&A]
Hello,

I wonder if it is possible to add filters to determine the automatic teams, for example, based by standing, by date of connection, or by date of arrival on the seat, or possibly by the different Customize profile button Sign In page?
My goal is to be able to separate the applicants or the positive standings, and the active ones.

thank you in advance

**Comment by warlof** [comment]
For which data do you want filters? Is it on the api? In such case, which endpoint? By teams, are you talking about squads?

**Reply by gortador** [reply]
I would like, if possible, to add option filters in the automatic squad filters

Last connection to the seat (per number of days) not need API
Date of first connection (Date of first connection) not need api
By standings (-5 / -10 / 0 / +5 / +10) (need api standings)
By the seat connection button (single authentication option, if it is possible to know from which unique authentication button he accessed the seat)

If is possible

(Sorry for my English I am French speaking)
